### ISSUE TITLE
change speed of players

### ISSUE THREAD

**Issue by darchon7** [zeddidragon/sgott#13]
is there a way to change the speed of the players through this?

**Comment by FIRE-AKA-DRAZN** [comment]
I don't believe there is, no.  There is the possibility it could do, if you can find the .SGO object for each player class.  However none of our rebalancing tools currently touch anything other than weapons and vehicles.
